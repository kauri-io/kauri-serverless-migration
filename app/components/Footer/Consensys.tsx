import React from 'react'
import styled from 'styled-components'

const Container = styled.div`
    display: flex;
    flex-direction: row;
    align-items: center;
    > :nth-child(2) {
        margin-left: 15px;
    }
    > :last-child {
        margin-left: 4px;
    }
`

const ConsenSysFormationIcon = styled.img`
    width: 34px;
`

const ConsenSysLogoIcon = styled.a`
    color: #fff;
    font-weight: 500;
    font-size: 13px;
`

const PrivacyPolicy = styled(ConsenSysLogoIcon)`
    margin-left: 10px;
    margin-right: 3px;
    font-weight: normal;
    font-size: 10px;
`

const TermsOfService = styled(PrivacyPolicy)`
    margin-left: 0px;
`

export default () => (
    <Container>
        <ConsenSysLogoIcon>Â© Copyright 2019</ConsenSysLogoIcon>
        <ConsenSysFormationIcon
            src={'/static/images/icons/footer-consensys.png'}
        />
        <ConsenSysLogoIcon>ConsenSys Formation</ConsenSysLogoIcon>
        <PrivacyPolicy href="/privacy-policy">Privacy Policy </PrivacyPolicy>
        <ConsenSysLogoIcon>|</ConsenSysLogoIcon>
        <TermsOfService href="/terms-of-use">Terms of Use</TermsOfService>
    </Container>
)
