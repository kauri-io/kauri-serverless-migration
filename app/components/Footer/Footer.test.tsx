import Footer from './index'
import { shallowWithTheme } from '../../setupTests'

describe('components/Footer', () => {
    it('snapshot should match', () => {
        const wrapper = shallowWithTheme(<Footer />)
        expect(wrapper).toMatchSnapshot()
    })
})
