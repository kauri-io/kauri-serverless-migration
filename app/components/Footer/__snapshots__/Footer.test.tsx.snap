// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Footer should match snapshot 1`] = `
<ThemeProvider
  theme={
    Object {
      "breakpoints": Object {
        "between": [Function],
        "down": [Function],
        "keys": Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ],
        "only": [Function],
        "up": [Function],
        "values": Object {
          "lg": 1280,
          "md": 960,
          "sm": 600,
          "xl": 1920,
          "xs": 0,
        },
        "width": [Function],
      },
      "direction": "ltr",
      "mixins": Object {
        "gutters": [Function],
        "toolbar": Object {
          "@media (min-width:0px) and (orientation: landscape)": Object {
            "minHeight": 48,
          },
          "@media (min-width:600px)": Object {
            "minHeight": 64,
          },
          "minHeight": 56,
        },
      },
      "overrides": Object {},
      "palette": Object {
        "action": Object {
          "active": "rgba(0, 0, 0, 0.54)",
          "disabled": "rgba(0, 0, 0, 0.26)",
          "disabledBackground": "rgba(0, 0, 0, 0.12)",
          "hover": "rgba(0, 0, 0, 0.08)",
          "hoverOpacity": 0.08,
          "selected": "rgba(0, 0, 0, 0.14)",
        },
        "augmentColor": [Function],
        "background": Object {
          "default": "#f5f5f5",
          "paper": "#fff",
        },
        "common": Object {
          "black": "#1E2428",
          "white": "#fff",
        },
        "contrastThreshold": 3,
        "divider": "rgba(0, 0, 0, 0.12)",
        "error": Object {
          "contrastText": "#fff",
          "dark": "#d32f2f",
          "light": "#e57373",
          "main": "#f44336",
        },
        "getContrastText": [Function],
        "grey": Object {
          "100": "#f5f5f5",
          "200": "#eeeeee",
          "300": "#e0e0e0",
          "400": "#bdbdbd",
          "50": "#fafafa",
          "500": "#9e9e9e",
          "600": "#757575",
          "700": "#616161",
          "800": "#424242",
          "900": "#212121",
          "A100": "#d5d5d5",
          "A200": "#aaaaaa",
          "A400": "#303030",
          "A700": "#616161",
        },
        "primary": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "rgb(7, 118, 93)",
          "light": "rgb(59, 186, 158)",
          "main": "#0BA986",
        },
        "secondary": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "rgb(178, 178, 178)",
          "light": "rgb(255, 255, 255)",
          "main": "#ffffff",
        },
        "text": Object {
          "disabled": "rgba(0, 0, 0, 0.38)",
          "hint": "rgba(0, 0, 0, 0.38)",
          "primary": "rgba(0, 0, 0, 0.87)",
          "secondary": "rgba(0, 0, 0, 0.54)",
        },
        "tonalOffset": 0.2,
        "type": "light",
      },
      "props": Object {},
      "shadows": Array [
        "none",
        "0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12)",
        "0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12)",
        "0px 3px 3px -2px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 1px 8px 0px rgba(0,0,0,0.12)",
        "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
        "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
        "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
        "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
        "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
        "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
        "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
        "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
        "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
        "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
        "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
        "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
        "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
        "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
        "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
        "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
        "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
        "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
        "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
        "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
        "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
      ],
      "shape": Object {
        "borderRadius": 4,
      },
      "spacing": [Function],
      "transitions": Object {
        "create": [Function],
        "duration": Object {
          "complex": 375,
          "enteringScreen": 225,
          "leavingScreen": 195,
          "short": 250,
          "shorter": 200,
          "shortest": 150,
          "standard": 300,
        },
        "easing": Object {
          "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
          "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
          "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
          "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
        },
        "getAutoHeightDuration": [Function],
      },
      "typography": Object {
        "body1": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1rem",
          "fontWeight": 400,
          "letterSpacing": "0.00938em",
          "lineHeight": 1.5,
        },
        "body2": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.875rem",
          "fontWeight": 400,
          "letterSpacing": "0.01071em",
          "lineHeight": 1.43,
        },
        "button": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.875rem",
          "fontWeight": 500,
          "letterSpacing": "0.02857em",
          "lineHeight": 1.75,
          "textTransform": "uppercase",
        },
        "caption": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.75rem",
          "fontWeight": 400,
          "letterSpacing": "0.03333em",
          "lineHeight": 1.66,
        },
        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
        "fontSize": 14,
        "fontWeightBold": 700,
        "fontWeightLight": 300,
        "fontWeightMedium": 500,
        "fontWeightRegular": 400,
        "h1": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "6rem",
          "fontWeight": 300,
          "letterSpacing": "-0.01562em",
          "lineHeight": 1,
        },
        "h2": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "3.75rem",
          "fontWeight": 300,
          "letterSpacing": "-0.00833em",
          "lineHeight": 1,
        },
        "h3": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "3rem",
          "fontWeight": 400,
          "letterSpacing": "0em",
          "lineHeight": 1.04,
        },
        "h4": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "2.125rem",
          "fontWeight": 400,
          "letterSpacing": "0.00735em",
          "lineHeight": 1.17,
        },
        "h5": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1.5rem",
          "fontWeight": 400,
          "letterSpacing": "0em",
          "lineHeight": 1.33,
        },
        "h6": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1.25rem",
          "fontWeight": 500,
          "letterSpacing": "0.0075em",
          "lineHeight": 1.6,
        },
        "htmlFontSize": 16,
        "overline": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.75rem",
          "fontWeight": 400,
          "letterSpacing": "0.08333em",
          "lineHeight": 2.66,
          "textTransform": "uppercase",
        },
        "pxToRem": [Function],
        "round": [Function],
        "subtitle1": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1rem",
          "fontWeight": 400,
          "letterSpacing": "0.00938em",
          "lineHeight": 1.75,
        },
        "subtitle2": Object {
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.875rem",
          "fontWeight": 500,
          "letterSpacing": "0.00714em",
          "lineHeight": 1.57,
        },
      },
      "zIndex": Object {
        "appBar": 1100,
        "drawer": 1200,
        "mobileStepper": 1000,
        "modal": 1300,
        "snackbar": 1400,
        "speedDial": 1050,
        "tooltip": 1500,
      },
      Symbol(mui.nested): false,
    }
  }
>
  <ThemeProvider
    theme={
      Object {
        "DEFAULT_CARD_WIDTH": 288,
        "MAIN_BODY_WIDTH": 1242,
        "bg": Object {
          "bgPrimary": "#1E2428",
          "bgSecondary": "#1E3D3B",
          "borderTextArea": "#979797",
          "contentBorder": "1px solid rgba(30, 36, 40, 0.19)",
          "disabledBackgroundColor": "#D6D6D6",
          "disabledTextColor": "#9B9B9B",
          "divider": "#EBEBEB",
          "errorRedColor": "#C03030",
          "hoverTextColor": "#108E72",
          "padding": "calc((100vw - \${props => props.theme.breakpoints[2]}) / 2)",
          "paddingTop": "2.5em",
          "primary": "#0BA986",
          "primaryColor": "#0BA986",
          "primaryDark": "#11856B",
          "primaryTextColor": "#1E2428",
          "secondary": "#11856B",
          "secondaryBlue": "rgb(81, 123, 162, 1)",
          "secondaryBlueDark": "rgb(81, 118, 142, 1)",
          "secondaryColor": "#1E3D3B",
          "secondaryTextColor": "#283035",
          "tertiaryBackgroundColor": "#F7F7F7",
          "textPrimary": "#1E2428",
          "white": "#FFFFFF",
        },
        "bgPrimary": "#1E2428",
        "bgSecondary": "#1E3D3B",
        "borderTextArea": "#979797",
        "breakpoints": Array [
          "500px",
          "52em",
          "1242px",
        ],
        "colors": Object {
          "bgPrimary": "#1E2428",
          "bgSecondary": "#1E3D3B",
          "borderTextArea": "#979797",
          "contentBorder": "1px solid rgba(30, 36, 40, 0.19)",
          "disabledBackgroundColor": "#D6D6D6",
          "disabledTextColor": "#9B9B9B",
          "divider": "#EBEBEB",
          "errorRedColor": "#C03030",
          "hoverTextColor": "#108E72",
          "padding": "calc((100vw - \${props => props.theme.breakpoints[2]}) / 2)",
          "paddingTop": "2.5em",
          "primary": "#0BA986",
          "primaryColor": "#0BA986",
          "primaryDark": "#11856B",
          "primaryTextColor": "#1E2428",
          "secondary": "#11856B",
          "secondaryBlue": "rgb(81, 123, 162, 1)",
          "secondaryBlueDark": "rgb(81, 118, 142, 1)",
          "secondaryColor": "#1E3D3B",
          "secondaryTextColor": "#283035",
          "tertiaryBackgroundColor": "#F7F7F7",
          "textPrimary": "#1E2428",
          "white": "#FFFFFF",
        },
        "contentBorder": "1px solid rgba(30, 36, 40, 0.19)",
        "disabledBackgroundColor": "#D6D6D6",
        "disabledTextColor": "#9B9B9B",
        "divider": "#EBEBEB",
        "errorRedColor": "#C03030",
        "fontSizes": Array [
          11,
          13,
          14,
          16,
          18,
          20,
          22,
          28,
          32,
        ],
        "fontWeight": Array [
          300,
          "normal",
          500,
          700,
        ],
        "hoverTextColor": "#108E72",
        "padContent": "
    max-width: 1242px;
    margin-left: auto;
    margin-right: auto;
    @media (min-width: 1242px) {
      min-width: 1242px;
    }
    @media (max-width: 1242px) {
      min-width: 90%;
    }
  ",
        "padding": "calc((100vw - 1242px) / 2)",
        "paddingTop": "2.5em",
        "primary": "#0BA986",
        "primaryColor": "#0BA986",
        "primaryDark": "#11856B",
        "primaryTextColor": "#1E2428",
        "secondary": "#11856B",
        "secondaryBlue": "rgb(81, 123, 162, 1)",
        "secondaryBlueDark": "rgb(81, 118, 142, 1)",
        "secondaryColor": "#1E3D3B",
        "secondaryTextColor": "#283035",
        "space": Array [
          0,
          10,
          15,
          30,
          60,
        ],
        "tertiaryBackgroundColor": "#F7F7F7",
        "textPrimary": "#1E2428",
        "white": "#FFFFFF",
      }
    }
  >
    <StyledFooter>
      <Footer__NewFooter>
        <StyledComponent
          forwardedComponent={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "attrs": Array [],
              "componentStyle": ComponentStyle {
                "componentId": "Footer__NewFooter-wussxj-0",
                "isStatic": false,
                "lastClassName": "fAOHvO",
                "rules": Array [
                  "display:grid;grid-template-columns:1fr 2fr 1fr;grid-auto-columns:auto;width:100%;background-color:",
                  [Function],
                  ";padding:0px ",
                  [Function],
                  ";@media (max-width:500px){display:none;}",
                ],
              },
              "displayName": "Footer__NewFooter",
              "foldedComponentIds": Array [],
              "render": [Function],
              "styledComponentId": "Footer__NewFooter-wussxj-0",
              "target": "div",
              "toString": [Function],
              "usesTheme": true,
              "warnTooManyClasses": [Function],
              "withComponent": [Function],
            }
          }
          forwardedRef={null}
        >
          <div
            className="Footer__NewFooter-wussxj-0 fAOHvO"
          >
            <_default>
              <Email__Container>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "Email__Container-nxd7qh-0",
                        "isStatic": false,
                        "lastClassName": "bkJUoo",
                        "rules": Array [
                          "display:flex;flex-direction:row;align-items:center;",
                        ],
                      },
                      "displayName": "Email__Container",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "Email__Container-nxd7qh-0",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="Email__Container-nxd7qh-0 bkJUoo"
                  >
                    <Email__EmailAddress
                      href="mailto:info@kauri.io"
                    >
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Email__EmailAddress-nxd7qh-1",
                              "isStatic": false,
                              "lastClassName": "jReXyi",
                              "rules": Array [
                                "height:16px;width:82px;color:#0ba986;font-family:Roboto;font-size:12px;font-weight:bold;line-height:16px;",
                              ],
                            },
                            "displayName": "Email__EmailAddress",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Email__EmailAddress-nxd7qh-1",
                            "target": "a",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                        href="mailto:info@kauri.io"
                      >
                        <a
                          className="Email__EmailAddress-nxd7qh-1 jReXyi"
                          href="mailto:info@kauri.io"
                        >
                          info@kauri.io
                        </a>
                      </StyledComponent>
                    </Email__EmailAddress>
                  </div>
                </StyledComponent>
              </Email__Container>
            </_default>
            <_default>
              <Logo__Container>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "Logo__Container-yhmxu8-1",
                        "isStatic": false,
                        "lastClassName": "kPuFaV",
                        "rules": Array [
                          "display:flex;align-self:center;flex-direction:column;align-items:center;margin:auto;padding:10px;",
                        ],
                      },
                      "displayName": "Logo__Container",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "Logo__Container-yhmxu8-1",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="Logo__Container-yhmxu8-1 kPuFaV"
                  >
                    <Logo__LogoIcon
                      src="/static/images/icons/footer-logo.png"
                    >
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Logo__LogoIcon-yhmxu8-0",
                              "isStatic": false,
                              "lastClassName": "kQopY",
                              "rules": Array [
                                "width:26px;height:25px;",
                              ],
                            },
                            "displayName": "Logo__LogoIcon",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Logo__LogoIcon-yhmxu8-0",
                            "target": "img",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                        src="/static/images/icons/footer-logo.png"
                      >
                        <img
                          className="Logo__LogoIcon-yhmxu8-0 kQopY"
                          src="/static/images/icons/footer-logo.png"
                        />
                      </StyledComponent>
                    </Logo__LogoIcon>
                    <Logo__KauriCopy>
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Logo__KauriCopy-yhmxu8-2",
                              "isStatic": false,
                              "lastClassName": "kdcflv",
                              "rules": Array [
                                "color:#fff;line-height:12px;font-size:12px;",
                              ],
                            },
                            "displayName": "Logo__KauriCopy",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Logo__KauriCopy-yhmxu8-2",
                            "target": "span",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                      >
                        <span
                          className="Logo__KauriCopy-yhmxu8-2 kdcflv"
                        >
                          kauri
                        </span>
                      </StyledComponent>
                    </Logo__KauriCopy>
                  </div>
                </StyledComponent>
              </Logo__Container>
            </_default>
            <_default>
              <Consensys__Container>
                <StyledComponent
                  forwardedComponent={
                    Object {
                      "$$typeof": Symbol(react.forward_ref),
                      "attrs": Array [],
                      "componentStyle": ComponentStyle {
                        "componentId": "Consensys__Container-sc-3l6ajh-0",
                        "isStatic": false,
                        "lastClassName": "hzIdEE",
                        "rules": Array [
                          "display:flex;flex-direction:row;align-items:center;>:nth-child(2){margin-left:15px;}>:last-child{margin-left:4px;}",
                        ],
                      },
                      "displayName": "Consensys__Container",
                      "foldedComponentIds": Array [],
                      "render": [Function],
                      "styledComponentId": "Consensys__Container-sc-3l6ajh-0",
                      "target": "div",
                      "toString": [Function],
                      "usesTheme": false,
                      "warnTooManyClasses": [Function],
                      "withComponent": [Function],
                    }
                  }
                  forwardedRef={null}
                >
                  <div
                    className="Consensys__Container-sc-3l6ajh-0 hzIdEE"
                  >
                    <Consensys__ConsenSysLogoIcon>
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                              "isStatic": false,
                              "lastClassName": "iJDtiZ",
                              "rules": Array [
                                "color:#fff;font-weight:500;font-size:13px;",
                              ],
                            },
                            "displayName": "Consensys__ConsenSysLogoIcon",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                            "target": "a",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                      >
                        <a
                          className="Consensys__ConsenSysLogoIcon-sc-3l6ajh-2 iJDtiZ"
                        >
                          © Copyright 2019
                        </a>
                      </StyledComponent>
                    </Consensys__ConsenSysLogoIcon>
                    <Consensys__ConsenSysFormationIcon
                      src="/static/images/icons/footer-consensys.png"
                    >
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Consensys__ConsenSysFormationIcon-sc-3l6ajh-1",
                              "isStatic": false,
                              "lastClassName": "gRtYkn",
                              "rules": Array [
                                "width:34px;",
                              ],
                            },
                            "displayName": "Consensys__ConsenSysFormationIcon",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Consensys__ConsenSysFormationIcon-sc-3l6ajh-1",
                            "target": "img",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                        src="/static/images/icons/footer-consensys.png"
                      >
                        <img
                          className="Consensys__ConsenSysFormationIcon-sc-3l6ajh-1 gRtYkn"
                          src="/static/images/icons/footer-consensys.png"
                        />
                      </StyledComponent>
                    </Consensys__ConsenSysFormationIcon>
                    <Consensys__ConsenSysLogoIcon>
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                              "isStatic": false,
                              "lastClassName": "iJDtiZ",
                              "rules": Array [
                                "color:#fff;font-weight:500;font-size:13px;",
                              ],
                            },
                            "displayName": "Consensys__ConsenSysLogoIcon",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                            "target": "a",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                      >
                        <a
                          className="Consensys__ConsenSysLogoIcon-sc-3l6ajh-2 iJDtiZ"
                        >
                          ConsenSys Formation
                        </a>
                      </StyledComponent>
                    </Consensys__ConsenSysLogoIcon>
                    <Consensys__PrivacyPolicy
                      href="/privacy-policy"
                    >
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Consensys__PrivacyPolicy-sc-3l6ajh-3",
                              "isStatic": false,
                              "lastClassName": "hDPWys",
                              "rules": Array [
                                "color:#fff;font-weight:500;font-size:13px;",
                                "margin-left:10px;margin-right:3px;font-weight:normal;font-size:10px;",
                              ],
                            },
                            "displayName": "Consensys__PrivacyPolicy",
                            "foldedComponentIds": Array [
                              "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                            ],
                            "render": [Function],
                            "styledComponentId": "Consensys__PrivacyPolicy-sc-3l6ajh-3",
                            "target": "a",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                        href="/privacy-policy"
                      >
                        <a
                          className="Consensys__ConsenSysLogoIcon-sc-3l6ajh-2 Consensys__PrivacyPolicy-sc-3l6ajh-3 hDPWys"
                          href="/privacy-policy"
                        >
                          Privacy Policy 
                        </a>
                      </StyledComponent>
                    </Consensys__PrivacyPolicy>
                    <Consensys__ConsenSysLogoIcon>
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                              "isStatic": false,
                              "lastClassName": "iJDtiZ",
                              "rules": Array [
                                "color:#fff;font-weight:500;font-size:13px;",
                              ],
                            },
                            "displayName": "Consensys__ConsenSysLogoIcon",
                            "foldedComponentIds": Array [],
                            "render": [Function],
                            "styledComponentId": "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                            "target": "a",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                      >
                        <a
                          className="Consensys__ConsenSysLogoIcon-sc-3l6ajh-2 iJDtiZ"
                        >
                          |
                        </a>
                      </StyledComponent>
                    </Consensys__ConsenSysLogoIcon>
                    <Consensys__TermsOfService
                      href="/terms-of-use"
                    >
                      <StyledComponent
                        forwardedComponent={
                          Object {
                            "$$typeof": Symbol(react.forward_ref),
                            "attrs": Array [],
                            "componentStyle": ComponentStyle {
                              "componentId": "Consensys__TermsOfService-sc-3l6ajh-4",
                              "isStatic": false,
                              "lastClassName": "hrfJKX",
                              "rules": Array [
                                "color:#fff;font-weight:500;font-size:13px;",
                                "margin-left:10px;margin-right:3px;font-weight:normal;font-size:10px;",
                                "margin-left:0px;",
                              ],
                            },
                            "displayName": "Consensys__TermsOfService",
                            "foldedComponentIds": Array [
                              "Consensys__ConsenSysLogoIcon-sc-3l6ajh-2",
                              "Consensys__PrivacyPolicy-sc-3l6ajh-3",
                            ],
                            "render": [Function],
                            "styledComponentId": "Consensys__TermsOfService-sc-3l6ajh-4",
                            "target": "a",
                            "toString": [Function],
                            "usesTheme": false,
                            "warnTooManyClasses": [Function],
                            "withComponent": [Function],
                          }
                        }
                        forwardedRef={null}
                        href="/terms-of-use"
                      >
                        <a
                          className="Consensys__ConsenSysLogoIcon-sc-3l6ajh-2 Consensys__PrivacyPolicy-sc-3l6ajh-3 Consensys__TermsOfService-sc-3l6ajh-4 hrfJKX"
                          href="/terms-of-use"
                        >
                          Terms of Use
                        </a>
                      </StyledComponent>
                    </Consensys__TermsOfService>
                  </div>
                </StyledComponent>
              </Consensys__Container>
            </_default>
          </div>
        </StyledComponent>
      </Footer__NewFooter>
    </StyledFooter>
  </ThemeProvider>
</ThemeProvider>
`;
